src/components/ProductDetailAddToCart/ProductDetailAddToCart.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "material-ui/styles";
+import ButtonBase from "material-ui/ButtonBase";
+import Grid from "material-ui/Grid";
+import InputAdornment from "material-ui/Input/InputAdornment";
+import TextField from "material-ui/TextField";
+import Typography from "material-ui/Typography";
+import Minus from "mdi-material-ui/Minus";
+import Plus from "mdi-material-ui/Plus";
+import { inject, observer } from "mobx-react";
+import Divider from "components/Divider";
+
+const styles = (theme) => ({
+  addToCartButton: {
+    "padding": theme.spacing.unit,
+    "borderRadius": "2px",
+    "backgroundColor": theme.palette.primary.light,
+    "minWidth": "66%",
+    "&:hover": {
+      borderColor: theme.palette.reaction.activeElementBorderColor
+    },
+    "&:focus": {
+      outline: "auto 5px -webkit-focus-ring-color"
+    }
+  },
+  addToCartText: {
+    color: theme.palette.primary.contrastText,
+    fontWeight: 600
+  },
+  incrementButton: {
+    backgroundColor: theme.palette.reaction.activeElementBackground,
+    padding: 6
+  },
+  quantityContainer: {
+    padding: 0,
+    border: `1px solid ${theme.palette.reaction.borderColor}`,
+    backgroundColor: theme.palette.common.white,
+    borderRadius: 2
+  },
+  quantityGrid: {
+    marginBottom: theme.spacing.unit * 3
+  },
+  quantityInput: {
+    "width": "40px",
+    "textAlign": "center",
+    "&:focus": {
+      borderColor: "#80bdff",
+      boxShadow: "0 0 0 0.2rem rgba(0,123,255,.25)"
+    },
+    "borderLeft": `1px solid ${theme.palette.reaction.borderColor}`,
+    "borderRight": `1px solid ${theme.palette.reaction.borderColor}`
+  },
+  quantityTypography: {
+    marginBottom: theme.spacing.unit * 2
+  }
+});
+
+
+@withStyles(styles, { withTheme: true })
```

Comment:
`{ withTheme: true }` is not necessary is this case, because the theme prop is not being used in the component and `@withStyles` will automatically proved the prop in the styles object.